app/components/containerInspect/containerInspectController.js

```diff
@@ -1,12 +1,16 @@
-angular.module('containerInspect', [])
+angular.module('containerInspect', ['angular-json-tree'])
 .controller('ContainerInspectController', ['$scope', '$transition$', 'Notifications', 'ContainerService',
 function ($scope, $transition$, Notifications, ContainerService) {
+
+  $scope.state = { DisplayTextView: false };
+  $scope.containerInfo = { object: {} };
```

Comment:
If you declare `$scope.containerInfo = {};` here, you'll be able to use `$scope.containerInfo = d;` below.

Reply:
That was it! Thank you.